src/v2/Components/Inquiry/config.ts

```diff
@@ -109,71 +112,95 @@ export const useEngine = () => {
+              !!context.current.phone &&
+              !!context.current.shareFollows &&
+              // And has logged all the relevant steps
+              (context.current.collectorLevel ?? 0) >= 3
+              ? // If you're a collector then you should have seen all of these
+                // before "completing" your profile
+                logger.current.hasLogged(
+                  "CommercialInterest",
+                  "ArtistsInCollection",
+                  "GalleriesYouWorkWith",
+                  "AuctionHousesYouWorkWith",
+                  "FairsYouAttend",
+                  "InstitutionalAffiliations"
+                )
+              : // If you've never bought art then you only saw this step
+                logger.current.hasLogged("CommercialInterest")
+          )
         },
-        hasSeenCommercialInterest: context => {
-          // TODO:
-          return false
+        hasSeenArtistsInCollection: () => {
+          return logger.current.hasLogged("ArtistsInCollection")
         },
-        hasSeenConfirmationThisSession: context => {
-          // TODO:
-          return false
+        hasSeenAuctionHousesYouWorkWith: () => {
+          return logger.current.hasLogged("AuctionHousesYouWorkWith")
         },
-        hasSeenFairsYouAttend: context => {
-          // TODO:
-          return false
+        hasSeenCommercialInterest: () => {
+          return logger.current.hasLogged("CommercialInterest")
         },
-        hasSeenGalleriesYouWorkWith: context => {
-          // TODO:
-          return false
+        hasSeenConfirmationThisSession: () => {
+          return logger.current.hasLogged("ConfirmationThisSession")
         },
-        hasSeenInstitutionalAffiliations: context => {
-          // TODO:
-          return false
+        hasSeenFairsYouAttend: () => {
+          return logger.current.hasLogged("FairsYouAttend")
         },
-        helpBy: context => {
-          // TODO:
-          return false
+        hasSeenGalleriesYouWorkWith: () => {
+          return logger.current.hasLogged("GalleriesYouWorkWith")
         },
-        isCollector: context => {
-          // TODO:
-          return false
+        hasSeenInstitutionalAffiliations: () => {
+          return logger.current.hasLogged("InstitutionalAffiliations")
         },
-        isLoggedIn: context => {
-          return !!context.isLoggedIn
+        isCollector: () => {
+          return (context.current?.collectorLevel ?? 0) >= 3
         },
-        isLoggedOut: context => {
-          return !context.isLoggedIn
+        isLoggedOut: () => {
+          return !context.current?.isLoggedIn
         },
       },
     })
-  }, [isLoggedIn, user])
+  )
 
-  const [current, setCurrent] = useState(engine.current())
+  const [current, setCurrent] = useState(engine.current.current())
 
   const View = VIEWS[current as View]
 
+  // Log each step as it updates
+  useEffect(() => {
+    logger.current.log(current)
```

Comment:
Should we log outside of `NODE_ENV=development`?

Reply:
This is a very different kind of logger; I'll just rename it. Basically it's just a thing that keeps track of what views you've seen and persists between sessions.